### ISSUE TITLE
Specify external partition by block device

### ISSUE THREAD

**Issue by krichter722** [SpazeDog/mounts2sd#28]
As far as I understood, mounts2sd uses a partition (or two if swap is enabled) based on convention and fails quiet silently if it doesn't find it. Has there been discussion about specifying the partition to use with a path to a block device or a UUID (I don't know if Android has /dev/by-uuid/ mapping). This would not only allow to use specific partitions (rather than "the largest"), but also to get an immediate feedback when setting the partition in the settings dialog whether it can be used or not (and in this case of course why :)). This would also cover issues like https://github.com/SpazeDog/mounts2sd/issues/26.

**Comment by jelabarre59** [comment]
It's especially a problem on shoddily-designed products like the Nabi2.  The Nabi2 uses /sdcard2 as a link to one of it's own internal partitions.  A lot of apps will presume the highest-numbered sdcard# is the external, while that is not the case with the Nabi2 (which puts external storage at /sdcard1).  Desperately trying to find a way to make this kids tablet usable for something other than the shoddy built-in apps.

**Comment by ildar** [comment]
> rather than "the largest"

wrong. It uses exactly the primary partition # 2. If it doesn't work, you should have attached logs.
